test/metabase/lib/convert_test.cljc

```diff
@@ -1060,21 +1191,3 @@
         (testing "round trip to pMBQL and back with small changes"
           (is (= query
                  (lib.convert/->legacy-MBQL (lib.convert/->pMBQL query)))))))))
-
-(deftest ^:parallel round-trip-expression-literal-test
```

Comment:
I just moved and renamed this above to be next to the existing similar roundtrip test already in this branch.